src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -69,13 +63,6 @@ network.host: "{{ lookup('dig', inventory_hostname, wantlist=True)[0] }}"
 indices.fielddata.cache.size:  {{ elasticsearch_fielddata_cache_size }}
 {% endif %}
 
-{% if elasticsearch_enable_inline_groovy_scripts %}
-{% if elasticsearch_version is version('5.6.16', '<=') %}
```

Comment:
Just going off of this, these settings seemed to also apply to ES5 and therefore aren't strictly an ES2 setting? Are they still safe to remove?

Reply:
Yes this is safe to remove and is intentionally removed. This was there to in ES 5 to support if someone was using groovy scripts which are deprecated. 
If someone installs commcare now they will not need es5 with groovy scripts so it is safe to remove for ES5 as well.